esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3719,7 +3719,7 @@ void CWsWuFileHelper::createWUZAPFile(IEspContext& context, Owned<IConstWorkUnit
     createZAPWUGraphProgressFile(request.wuid.str(), inFileNamePrefixWithPath.str());
     createProcessLogfile(cwu, winfo, "EclAgent", folderToZIP.str());
     createProcessLogfile(cwu, winfo, "Thor", folderToZIP.str());
-    if (strieq(request.includeThorSlaveLog.str(), "on"))
+    if (request.includeThorSlaveLog.isEmpty() && strieq(request.includeThorSlaveLog.str(), "on"))
```

Comment:
@wangkx I don't understand this Jira/PR.  Please add a commit message

Reply:
The request.includeThorSlaveLog contains the ZAP request option for IncludeThorSlaveLog. When it is empty (the option is not specified), the ZAP report should include thor slave log as default. @RussWhitehead

Reply:
It is my mistake. Should be || not && in case the option is not set (the request may not be from ECLWatch UI).